smartcontractkit/chainlink
# Node version exposed with Prometheus

Closes https://app.shortcut.com/chainlinklabs/story/12633/export-node-version-in-a-prometheus-label

I talked to @tyrion70 and he confirmed the version output format looks good:

```
# HELP version Node version information
# TYPE version counter
version{commit="5eb4f2a7bf52cbd4e0c2fb1848bbe2051f9d0268",version="0.10.14"} 1
```

How to test:
- start a CL node
- open `/metrics` path
- see the version information metric as shown in above
